### DISCUSSION TITLE
CSS Color Module Level 4 support

### DISCUSSION THREAD

**Discussion by lu10539** [Textualize/textual#2277] [Ideas]
I would like to add support to CSS Color Module Level 4 with new color spaces including LAB, LCH, OKLab, and OKLch. Chrome 111 supports Level 4 of the CSS Color Module. It's based on the D50 color space. [Link](https://developer.chrome.com/en/blog/new-in-chrome-111/#css-color-level4)
[CSS Color Module Level 4 for LAB, LCH, OKLab, and OKLch reference](https://www.w3.org/TR/css-color-4/#lab-colors)
[Can I Use CIE LAB? (CSS Color Level 4)](https://caniuse.com/?search=lab)
[CSS Color Guide for Chrome](https://developer.chrome.com/articles/high-definition-css-color-guide/#what-is-a-color-space)
[MDN CSS `lab` function](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/lab)

**Comment by willmcgugan** [comment]
This is not likely something we would prioritise. Given its not even supported by all browsers.

Are you saying you want to contribute support for these color spaces?
